libr/bin/format/elf/elf.c

```diff
@@ -2817,6 +2817,31 @@ RBinElfSection* Elf_(r_bin_elf_get_sections)(ELFOBJ *bin) {
 	return ret;
 }
 
+static bool is_special_arm_symbol(ELFOBJ *bin, Elf_(Sym) *sym, const char *name) {
+	if (name[0] != '$') {
+		return false;
+	}
+	switch (name[1]) {
```

Comment:
Can we improve this check? i think its a bit weak that way, didnt checked whats in name+2, to avoid false positives